### DISCUSSION TITLE
The bay between the western part of France and northern part of Spain is what?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#481] [Galaxy Trivia] [answered]
**Bonus:** What is the largest city in the province?

**Comment by ParanoidUser** [comment]
@wfrsk, @DurandSacha, @Malinowsk, @TheVigor, anyone? 😶‍🌫️

**Comment by thuongtruong1009** [comment, marked as answer]
The Bay of Biscay, which is bounded by the west coast of France and the north coast of Spain, covers an area of 86,000 sq miles. It's known for its rough seas and violent storms and much of this is thanks to its exposure to the Atlantic ocean.

**Reply by ParanoidUser** [reply]
Thank you, Tran Nguyen! Do you know the largest city in the bay?
